src/enochecker/enochecker.py

```diff
@@ -426,6 +426,10 @@ def run(self, method: Optional[str] = None) -> CheckerResult:
         try:
             ret = self._run_method(method)
             if ret is not None:
+                warnings.warn(
+                    "Returning a non-ok status code is not recommended and will be removed in the future. Raise an Exception with additional text instead.",
```

Comment:
"Returning a result is not recommended and will be removed in the future. Use EnoExceptions with additional text instead."